templates/repo/actions/workflow_dispatch.tmpl

```diff
@@ -0,0 +1,72 @@
+<div class="ui ignored info message project-column-header" >
+	<span class="ui left floated text middle">{{ctx.Locale.Tr "actions.workflow.has_workflow_dispatch"}}</span>
+	<button class="ui right floated mini button show-modal" data-modal="#runWorkflowDispatchModal">{{ctx.Locale.Tr "actions.workflow.run"}}{{svg "octicon-triangle-down" 14 "dropdown icon"}}</button>
+</div>
+<div id="runWorkflowDispatchModal" class="ui mini modal">
+	<div class="content">
+		<form id="runWorkflowDispatchForm" class="ui form" action="{{$.Link}}/run?workflow={{$.CurWorkflow}}&actor={{$.CurActor}}&status={{.Status}}" method="post">
+			{{.CsrfTokenHtml}}
+			<div class="ui inline field required">
+				<label>{{ctx.Locale.Tr "actions.workflow.from_ref"}} :</label>
+				<div class="ui floating dropdown icon button mini">
+					<input type="hidden" name="ref" value="refs/heads/{{index .Branches 0}}">
+					<i>{{svg "octicon-git-branch"}}</i>
+					<span class="text" style="font-size: medium">{{index .Branches 0}}</span>
+					<div class="menu">
+						<div class="ui icon search input">
+							<i class="search icon">{{svg "octicon-search" 16 "search icon"}}</i>
```

Comment:
```suggestion
							{{svg "octicon-search" 16 "search icon"}}
```

Reply:
`{{svg "octicon-search" 16 "search icon"}}': 
![image](https://github.com/go-gitea/gitea/assets/3114995/d4878a55-1e3f-4c2e-95a4-547921193f1a)

`<i class="search icon">{{svg "octicon-search" 16 "search icon"}}</i>`:
![image](https://github.com/go-gitea/gitea/assets/3114995/507737a6-8afe-41bf-92e0-f175381c9fbd)

I think the latter might be better

Reply:
Yes, look like a rare case where fomantic actually requires the `<i>`.